docs/src/main/sphinx/sql/merge.rst

```diff
@@ -93,5 +93,7 @@ table row::
 Limitations
 -----------
 
-Some connectors have limited or no support for ``MERGE``.
-See connector documentation for more details.
+While any connector can provide a ``MERGE`` source table, if a connector
+does not support ``MERGE`` statements it cannot be the target of a merge
+operation. See the :doc:`connector documentation </connector>` for more
+information.
```

Comment:
Is this better?

```rst
While any connector can be used as a source table for a ``MERGE`` statement not all connectors can
be the target of a merge operation. Only connectors which support the ``MERGE`` statement can be
the target of a merge operation. See the :doc:`connector documentation </connector>` for more
information.
```

Reply:
the 1st and 2nd sentences can be combined or reworded to something like:

```rst
Any connector can be used as a source table for a ``MERGE`` statement.
Only connectors which support the ``MERGE`` statement can be the target of a merge operation.
See the :doc:`connector documentation </connector>` for more
information.
```

Reply:
I like the second suggestion.. starting with "While" seems semi-ideal